src/lib/migration/main_workflow.rb

```diff
@@ -120,7 +125,22 @@ def perform_update
     end
 
     def create_snapshot
-      Yast2::FsSnapshot.create_single("before update on migration")
+      out = Yast::SCR.Execute(Yast::Path.new(".target.bash_output"), FIND_CONFIG_CMD)
+
+      log.info("Checking if Snapper is configured: \"#{FIND_CONFIG_CMD}\" returned: #{out}")
+      return :next unless out["exit"] == 0
+
+      cmd = format(CREATE_SNAPSHOT_CMD,
+        snapshot_type: :single,
+        description:   "before update on migration"
+      )
+
+      out = Yast::SCR.Execute(Yast::Path.new(".target.bash_output"), cmd)
+
+      if out["exit"] != 0
+        log.error "Snapshot could not be created: #{cmd} returned: #{out}"
+        Yast::Report.Error(_("Failed to create filesystem snapshot."))
```

Comment:
so user see this error message and he click ok and it goes further, so what is goal of this error message, what should user do?